frontend/javascripts/admin/api/certificate_validation.tsx

```diff
@@ -0,0 +1,19 @@
+import Request from "libs/request";
+
+type CertificateValidationResult = {
+  isValid: boolean;
+  expiresAt: number;
+};
+
+export async function isCertificateValid(): Promise<CertificateValidationResult> {
+  try {
+    return await Request.receiveJSON("/api/checkCertificate", { showErrorToast: false });
+  } catch (errorResponse: any) {
+    try {
```

Comment:
What happens here if the server is not reachable for a short period of time? In that case we shouldn't show the expiration modal.